tests/e2e/e2e_test.go

```diff
@@ -80,22 +80,24 @@ var _ = Describe("odoe2e", func() {
 	})
 
 	Context("odo utils config", func() {
-		It("should get true for updatenotification by default", func() {
+		It("should display all available settings and "+config.UpdateNotificationSetting+" should be true by default", func() {
 			configOutput := runCmd("odo utils config view")
 			Expect(configOutput).To(ContainSubstring("true"))
-			Expect(configOutput).To(ContainSubstring("UpdateNotification"))
+			Expect(configOutput).To(ContainSubstring(config.UpdateNotificationSetting))
+			Expect(configOutput).To(ContainSubstring(config.NamePrefixSetting))
+			Expect(configOutput).To(ContainSubstring(config.TimeoutSetting))
 		})
 		It("should be checking to see if timeout is the same as the constant", func() {
-			configOutput := runCmd("odo utils config view|grep Timeout")
+			configOutput := runCmd("odo utils config view|grep " + config.TimeoutSetting)
```

Comment:
Avoid using terminal specific command like ```grep```. regular expression would be an alternate choice and can be run on any terminal.

Reply:
I didn't create these tests, they've been like this for a while, not going to change them now.

Reply:
Ok, will track this change in #1129 then.